official/resnet/resnet_run_loop.py

```diff
@@ -384,16 +405,37 @@ def input_fn_eval():
       benchmark_logger = logger.BenchmarkLogger(flags.benchmark_log_dir)
       benchmark_logger.log_estimator_evaluation_result(eval_results)
 
+  # Return the classifier after all epochs are completed.
+  return classifier
+
+
+def export_savedmodel(classifier, export_dir, shape, batch_size):
+  """Exports a saved model for the given classifier."""
+  input_receiver_fn = export.build_tensor_serving_input_receiver_fn(
+      shape, batch_size=batch_size)
+  classifier.export_savedmodel(export_dir, input_receiver_fn)
+  return classifier
+
+
+def warn_on_multi_gpu_export(multi_gpu=False):
+  """For the time being, multi-GPU mode does not play nicely with exporting."""
+  if multi_gpu:
+    tf.logging.warning(
+        'You are exporting a SavedModel while in multi-GPU mode. Note that '
+        'the resulting SavedModel will require the same GPUs be available.'
+        'If you wish to serve the SavedModel from a different device, '
+        'try exporting the SavedModel with multi-GPU mode turned off.')
```

Comment:
I guess we should stay with double quote for consistence.

Reply:
This file uses single quotes. Perhaps sometime we should just choose which quote style to use for this repo.

Reply:
Yeah, I would opt for double quotes just because docstrings are `"""`, but we can fix in a separate PR for all files if desired. For now, will stick with what's there to avoid confusion in this PR.